src/main/java/com/netchill/webservices/api/GeneralWs.java

```diff
@@ -0,0 +1,35 @@
+package com.netchill.webservices.api;
+
+import com.coreoz.plume.jersey.security.permission.PublicApi;
+import com.netchill.db.generated.Genre;
+import com.netchill.services.configuration.ConfigurationService;
+import com.netchill.services.general.GeneralService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@Path("")
```

Comment:
la path n'est pas bon

```suggestion
@Path("/")
```